README.md

```diff
@@ -1,23 +1,28 @@
 # Certifications
 This repository will be used to house the Jakarta EE TCK Results for the Certification Requests.
 The TCK Results need to be in a publicly accessible website via a defined URL.
-These TCK Results do not need to be accessible from the main [openliberty.io](https://openliberty.io) site.
+These TCK Results are not indexed and, therefore, will not appear in search results.
+All TCK Results (all pages with `:page-layout: certification` to be exact) can be viewed on [openliberty.io/certifications/](https://openliberty.io/certifications/), which is also not indexed.
 
 ## File Format
-The TCK Results will be asciidoc (TCKResults.adoc) files.
+The TCK Results will be asciidoc (<runtimeVersion>TCKResults.adoc) files.
 This format will allow for easy rendering onto the openliberty.io site.
-Shortly after PRs are merged into this repo, the results will be accessible via https://openliberty.io/certifications/...
 
 ## Directory Structure
-The following directory structure will be used to house these TCK Results:
-
-```https://openliberty.io/certifications/jakartaee/<specification>/<specification version>/TCKResults```
-For example,
-```
-https://openliberty.io/certifications/jakartaee/platform/8/TCKResults.html
-https://openliberty.io/certifications/jakartaee/webprofile/8/TCKResults.html
-https://openliberty.io/certifications/jakartaee/servlet/4.0/TCKResults.html
-```
+The URL structure on [openliberty.io/certifications/](https://openliberty.io/certifications/) directly matches the directory structure of this repo.
+The following URL/path structure is to be used for this repo:
+
+```https://openliberty.io/certifications/<spec>/<specVersion>/<optional:subSpec>/<optional:subSpecVersion>/<runtimeVersion>-TCKResults```
+
+For example:
+> [certifications/jakartaee/8/webprofile/19.0.0.6-TCKResults.adoc](https://openliberty.io/certifications/jakartaee/8/webprofile/19.0.0.6-TCKResults.html)  
+> [certifications/jakartaee/9.1/platform/21.0.0.2-beta-Java11-TCKResults.adoc](https://openliberty.io/certifications/jakartaee/9.1/platform/21.0.0.2-beta-Java11-TCKResults.html)  
+>  
+> [certifications/microprofile/5.0/21.0.0.12-beta-TCKResults.adoc](https://openliberty.io/certifications/microprofile/5.0/21.0.0.12-beta-TCKResults.html)  
+> [certifications/microprofile/5.0/config/3.0/21.0.0.12-beta-TCKResults.adoc](https://openliberty.io/certifications/microprofile/5.0/config/3.0/21.0.0.12-beta-TCKResults.html)  
+>  
+> [certifications/microprofile/contextpropagation/ccr/1.2/21.0.0.3-beta-TCKResults.adoc](https://openliberty.io/certifications/microprofile/contextpropagation/ccr/1.2/21.0.0.3-beta-TCKResults.html)
```

Comment:
is the `ccr` part still needed?

Reply:
@Emily-Jiang ^

Reply:
don't need ccr any more

Reply:
ok, then I can update all the paths to get rid of ccr, along with the one that was just delivered for for MP CP 1.3.